devel/libcbor/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libcbor
+license             MIT License
+version             0.8.0
+revision            0
+categories          devel
+platforms           darwin
+maintainers         @trodemaster \
+                    openmaintainer
+description         library for parsing and generating CBOR
+PortGroup           cmake 1.1
+
+long_description    libcbor is a C library for parsing and generating CBOR, the general-purpose schema-less binary data format.
+homepage            https://github.com/PJK/libcbor
+master_sites        https://github.com/PJK/libcbor/archive/
+distfiles           v${version}.tar.gz
```

Comment:
so sorry -- please "github" Portgroup

something like this:
```
PortGroup github 1.0
github.setup PJK libcbor 0.8.0 v
```

Reply:
also goes at the top -- have a look at any other Portfile in the repo for inspiration. 1/2 of them use github.